packages/core/workflows-sdk/src/utils/composer/helpers/create-step-handler.ts

```diff
@@ -0,0 +1,83 @@
+import {CreateWorkflowComposerContext, StepExecutionContext} from "../type"
+import {WorkflowStepHandlerArguments} from "@medusajs/orchestration"
+import {resolveValue} from "./resolve-value"
+import {StepResponse} from "./step-response"
+import {deepCopy, OrchestrationUtils} from "@medusajs/utils"
+
+export function createStepHandler(
+  this: CreateWorkflowComposerContext,
+  { stepName, input, invokeFn, compensateFn }
```

Comment:
suggestion: I havent added the types here but we could wdyt?

Reply:
Done